corehq/apps/export/templates/export/download_data_files.html

```diff
@@ -0,0 +1,130 @@
+{% extends "style/base_section.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+
+{% block title %}{% trans "Download Data Files" %}{% endblock %}
+
+{% block page_content %}
+    {% block main_column_content %}
+
+<h2>{% trans "Download Data Files" %}</h2>
+
+{% if data_files %}
+<div class="row">
+    <div class="col-sm-9 col-md-8 col-lg-8">
+        <table class="table table-hover">
+            {% for data_file in data_files %}
+            <tr id="row_{{ data_file.id }}">
+                <td>
+                    <a href="{% url 'download_data_file' domain data_file.id data_file.filename %}">
+                        <i class="fa fa-file text-primary"></i>
+                        {{ data_file.filename }}
+                    </a>
+                </td>
+                <td>
+                    {{ data_file.description }}
+                    <textarea id="url_{{ data_file.id }}" style="display: none">{% if request.is_secure %}https://{% else %}http://{% endif %}{{ request.get_host }}{% url 'download_data_file' domain data_file.id data_file.filename %}</textarea>
```

Comment:
i think we usually use `dimagi.utils.web.get_url_base` for this unless this is different